weave/trace_server/remote_http_trace_server.py

```diff
@@ -67,7 +67,7 @@ def _is_retryable_exception(e: Exception) -> bool:
 
 
 def _log_retry(retry_state: tenacity.RetryCallState) -> None:
-    logger.info(
+    logger.warn(
```

Comment:
this by default prints out almost nothing useful. Can you please modify these such that the printed output will help us debug in the future?